src/components/vr/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
What does `vr` stand for? `vertical ruler`? Can you rename `vr` to a more descriptive name? I like all the other sub-directories in `components` directory.

Reply:
I made it just `<Vr/>` to match the built-in HTML `<hr/>` for horizontal rules that I use many places in the code.

I can change it to `<VerticalLine/>` but then I would also want to replace the `<hr/>`'s with `<HorizontalLine/>` to be consistent. I'll leave it to your preference

Reply:
It's perfectly fine to use `Vr` as the tag name in the code. I am suggesting a more descriptive name of directory that contains source code for this component.

Reply:
So far I've been following a convention of making sure to name the component the same as its folder name to avoid naming confusion. 

I think I'll just make them `<HorizontalLine/>` and `<VerticalLine/>`. I think `<hr>` wasn't a very good name choice when they made html, and if anyone comes back to edit this code later, they'll be able to understand "horizontalline" better.

Reply:
Thank you. That makes a lot of sense.